.docs/Scheduling-MacOS.md

```diff
@@ -200,19 +200,19 @@ Every Sunday at midnight and every Wednesday full hour (xx:00). Notice the inclu
 <key>StartCalendarInterval</key>
 <array>
         <dict>
-		<key>Weekday</key>
-		<integer>0</integer>
+  <key>Weekday</key>
+  <integer>0</integer>
                 <key>Hour</key>
                 <integer>00</integer>
                 <key>Minute</key>
                 <integer>00</integer>
         </dict>
-	<dict>
-		<key>Weekday</key>
-		<integer>3</integer>
-		<key>Minute</key>
-		<integer>00</integer>
-	</dict>
+ <dict>
+  <key>Weekday</key>
+  <integer>3</integer>
+  <key>Minute</key>
+  <integer>00</integer>
+ </dict>
 </array>
```

Comment:
Looks like indentation is messed up here. Can you please re-indent using 2 spaces for each level?